README.md

```diff
@@ -60,3 +60,18 @@ For `Stylelint`, you will need to add the following to your `.vscode/settings.js
 ## Schemas for Interactive Blocks and Content Templates
 
 Please refer to [schemas](docs/schemas.md) for details on using supported interactive capabilities.
+
+
+## Update the Events API Client Code in EventsAPI
+
+In order to update the client code generated by openapi to serve the EventsAPI you can use the command below. This process creates a docker container with java which runs the openapi-generator-cli commands. 
+
+If you want to modify the language or location of the client code, modify the arguments following -g or -o respectively. 
+
+```bash
+$ cd k12-apps-raise
+$ docker run --rm -v "${PWD}:/local" openapitools/openapi-generator-cli generate \
+  -i http://events.raiselearning.org/openapi.json \
```

Comment:
Super minor: I mean it'll work / redirect either way, but it would be nice to make this `https` vs `http`.